examples/graphql-react-apollo/src/ApolloClient.js

```diff
@@ -1,11 +1,18 @@
-import ApolloClient from 'apollo-boost'
+import { ApolloClient, InMemoryCache, createHttpLink } from '@apollo/client'
 
-// Isolate Apollo client so it could be reused
-// in both application runtime and tests.
-export const client = new ApolloClient({
+const cache = new InMemoryCache()
+
+const link = createHttpLink({
   uri: 'http://localhost:3000/graphql',
 
   // Use explicit `window.fetch` so tha outgoing requests
   // are captured and deferred until the Service Worker is ready.
   fetch: (...args) => fetch(...args),
```

Comment:
I noticed that changing this to `fetch: window.fetch` also worked, but I didn't want to make unnecessary changes. Just calling it out as a potential simplification
```suggestion
  fetch: window.fetch,
```